auditing/auditing-interceptor.go

```diff
@@ -73,7 +80,19 @@ func StreamServerInterceptor(a Auditing, logger *zap.SugaredLogger, shouldAudit
 		if !shouldAudit(info.FullMethod) {
 			return handler(srv, ss)
 		}
-		requestID := uuid.New().String()
+		var requestID string
+		if str, ok := ss.Context().Value(rest.RequestIDKey).(string); ok {
+			requestID = str
+		}
+		if requestID == "" {
+			requestID = uuid.New().String()
```

Comment:
here as well

Reply:
I made a global find & replace to catch the whole repo :+1: